.svglintrc.js

```diff
@@ -33,6 +34,19 @@ function sortObjectByValue(obj) {
     .reduce((r, k) => Object.assign(r, { [k]: obj[k] }), {});
 }
 
+/**
+ * Parses a SVG path using svg-path-segments library and building a svgpath
+ *   instance with the result, returning a convenient object with two properties:
+ *   - `segments`: array of path segments as are returned by svg-path-segments.
+ *   - `svgPath`: instance of svgpath object that can be manipulated easily.
+ **/
```

Comment:
I'm unfamiliar with this style of this style of documentation. Throughout the simple-icons project we use [JSDoc](https://jsdoc.app/)-style documentation, so I suggest we do the same here.

And one general guideline for writing this kind of documentation that I always try to go by is to describe only what the function does, not how it does that.

```suggestion
/**
 * Parses an SVG path into segments and builds an svgpath instance.
 *
 * @param iconPath The path value of the SVG.
 * @returns An object with two properties
 *     - segments Array of path segments as are returned by svg-path-segments.
  *     - svgPath Instance of svgpath object that can be manipulated easily.
 */
```